CVD_LIST.md

```diff
@@ -0,0 +1,42 @@
+# Coordinated Vulnerability Disclosure List
+
+This list is used to provide actionable information to multiple vendors at once.
+This list is not intended for individuals to find out about security issues.
+
+## Embargo Policy
+
+This policy forbids members of this project's [security contacts] and others
+defined below from sharing information outside of the [security contacts] and this
+listing without need-to-know and advance notice.
+
+The information members and others receive from the list defined below must:
+
+* not be made public,
+* not be shared,
+* not be hinted at,
+* must be kept confidential and close held
+
+Except with the list's explicit approval. This holds true until the public
+disclosure date/time that was agreed upon by the list.
+
+If information is inadvertently shared beyond what is allowed by this policy,
+you are REQUIRED to inform the [security contacts] of exactly what
+information leaked and to whom. A retrospective will take place after the leak
+so we can assess how to not make this mistake in the future.
+
+Violation of this policy will result in the immediate removal and subsequent
+replacement of you from this list or the [security contacts].
+
+## Membership Criteria
+
+To be eligible for joining the CVD list, you should:
+
+1. Have an actively monitored security email alias for your organization.
+2. Support active Flux instances to users beyond your own organization.
+3. Accept the [Embargo Policy](#embargo-policy) that is outlined above.
+
+[security contacts]: https://github.com/fluxcd/.github/blob/main/SECURITY.md
+
+### Request to Join
+
+Submit a request [here](https://docs.google.com/forms/d/1RJRUSwHUeKwJfZdcAzIkyKD4TeAweLLjXZkcN-uVDqA).
```

Comment:
How can I access the data from here, also how can I edit the form?

Reply:
Be a listed member in https://github.com/fluxcd/.github/blob/main/SECURITY.md.

Reply:
I am listed there but not 

We should add these details (such as access to this google doc form) to an onboarding/offboarding policy to the security team description. @dholbach mentions this here https://github.com/fluxcd/community/issues/214#issuecomment-1195262379 And in fluxcd/community/GOVERNANCE.md

Daniel and I discussed we should list `core maintainers: adding a security team member` to the supermajority decisions list here https://github.com/fluxcd/community/blob/main/GOVERNANCE.md#supermajority-decisions